.github/workflows/tests.yml

```diff
@@ -202,3 +202,65 @@ jobs:
         token: ${{ secrets.GITHUB_TOKEN }}
         label: ready-for-review
         type: add
+
+  changed-files:
+    runs-on: ubuntu-latest
+    outputs:
+      views: ${{steps.changes.outputs.views}}
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+    - name: Get changed files
+      id: changes
+      run:  |
+        echo "::set-output name=views::$(git diff --name-only --diff-filter=ACMRT ${{ github.event.pull_request.base.sha }} ${{ github.sha }} | grep app/views/ | xargs))"
+
+  translation-screenshot:
+    if: ${{ github.event.label.name == 'Translation System' }}
```

Comment:
This looks interesting! So, this is triggered only by that label. What gives it that label? 

My only caution here is that this extra script lengthens and complicates our standard test runs a bit. It might make it a little harder to read and maintain. For example, what if something changes in the test setup. We'd have to make sure this part also gets updated accordingly. But I'm not against it. Is there any way, though, to reduce the repetition and to rely on common reusable code?

Looking very good though!!